sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/package-info.java

```diff
@@ -2,7 +2,243 @@
+ *             LocalDate transactionDate = transactionDateField.getValue&#40;&#41;.asDate&#40;&#41;;
+ *             System.out.printf&#40;&quot;Transaction Date: %s, confidence: %.2f%n&quot;,
+ *                 transactionDate, transactionDateField.getConfidence&#40;&#41;&#41;;
+ *         &#125;
+ *     &#125;
+ *
+ *     FormField receiptItemsField = recognizedFields.get&#40;&quot;Items&quot;&#41;;
+ *     if &#40;receiptItemsField != null&#41; &#123;
+ *         System.out.printf&#40;&quot;Receipt Items: %n&quot;&#41;;
+ *         if &#40;FieldValueType.LIST == receiptItemsField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *             List&lt;FormField&gt; receiptItems = receiptItemsField.getValue&#40;&#41;.asList&#40;&#41;;
+ *             receiptItems.stream&#40;&#41;
+ *                 .filter&#40;receiptItem -&gt; FieldValueType.MAP == receiptItem.getValue&#40;&#41;.getValueType&#40;&#41;&#41;
+ *                 .map&#40;formField -&gt; formField.getValue&#40;&#41;.asMap&#40;&#41;&#41;
+ *                 .forEach&#40;formFieldMap -&gt; formFieldMap.forEach&#40;&#40;key, formField&#41; -&gt; &#123;
+ *                     if &#40;&quot;Quantity&quot;.equals&#40;key&#41;&#41; &#123;
+ *                         if &#40;FieldValueType.FLOAT == formField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *                             Float quantity = formField.getValue&#40;&#41;.asFloat&#40;&#41;;
+ *                             System.out.printf&#40;&quot;Quantity: %f, confidence: %.2f%n&quot;,
+ *                                 quantity, formField.getConfidence&#40;&#41;&#41;;
+ *                         &#125;
+ *                     &#125;
+ *                 &#125;&#41;&#41;;
+ *         &#125;
+ *     &#125;
+ * &#125;
+ * </pre>
+ * <!-- end readme-sample-recognize-receipt-url -->
+ *
+ * <p>
+ * You can also extract data from a local receipt with prebuilt models using the
+ * {@link com.azure.ai.formrecognizer.FormRecognizerClient#beginRecognizeReceipts(java.io.InputStream, long, com.azure.ai.formrecognizer.models.RecognizeReceiptsOptions, com.azure.core.util.Context)}
+ * method.</p>
+ *
+ * <p>For more information on which supported model you should use refer to
+ * <a href="https://learn.microsoft.com/azure/applied-ai-services/form-recognizer/concept-model-overview?view=form-recog-2.1.0">models usage
+ * documentation</a>.</p>
+ *
+ * <br/>
+ *
+ * <hr/>
+ *
+ * <h2>Analyze a custom form with a model trained with or without labels.</h2>
+ *
+ * Analyze a custom form with a model trained with or without labels. Custom models are trained with your own data,
+ * so they're tailored to your documents.
+ *
+ * <p>For more information, see
+ * <a href="https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/v3/TrainModelWithLabels.java">train a model with labels</a>.
```

Comment:
The link outs should be done via aka.ms links.
That allows the documentation to remain backward compatible if links change in future.

you can register your aka.ms links by going to aka.ms